web/packages/design/src/DataTable/useTable.ts

```diff
@@ -50,8 +51,8 @@ export default function useTable<T>({
     }
 
     return {
-      data: serversideProps || disableFilter ? data : [],
```

Comment:
Why are we getting rid of this behavior? It seems unrelated to client-side search.

Reply:
we have a [useEffect](https://github.com/gravitational/teleport/blob/564c245ecae10187371c1f8ab056093104ef5f31/web/packages/design/src/DataTable/useTable.ts#L192) that will set the `data`

before, it caused a brief flashing where the full list showed, until the searchValue got applied